src/analyzer/protocol/ssl/SSL.h

```diff
@@ -33,11 +34,24 @@ class SSL_Analyzer final : public analyzer::tcp::TCP_ApplicationAnalyzer {
 	static analyzer::Analyzer* Instantiate(Connection* conn)
 		{ return new SSL_Analyzer(conn); }
 
+	// Key material for decryption
```

Comment:
Since these are new functions, we should follow our general, more verbose documentation guidelines. See e.g. how the functions in https://github.com/zeek/zeek/blob/master/src/analyzer/Manager.h are documented.

Reply:
Yeah, documentation is definitely not there yet. I'll come up with something on the next pass.

Reply:
Addressed in the latest version